docker-compose.yml

```diff
@@ -755,17 +757,15 @@ services:
         base: ${R_ORG}/${R_IMAGE}:${R_TAG}
     shm_size: *shm-size
     environment:
-      TEST_R_WITH_ARROW: "true"
```

Comment:
Is this change desired? I admit I don't understand what all those environment variables are doing.

Reply:
Previously this job wasn't using `ci/scripts/r_test.sh` but now it is, and that env var is set in the script.